crates/ruff/src/resolve.rs

```diff
@@ -21,6 +21,9 @@ pub fn resolve(
     config_arguments: &ConfigArguments,
     stdin_filename: Option<&Path>,
 ) -> Result<PyprojectConfig> {
+    let Ok(_cwd) = std::env::current_dir() else {
```

Comment:
It feels off that the path isn't used here. I think we should change the code to call `cwd.parse_dot()` and use it instead of `&path_dedot::CWD` below